airflow/providers/google/firebase/example_dags/example_firestore.py

```diff
@@ -119,10 +119,6 @@
                 "csvOptions": {"skipLeadingRows": 1},
             },
         },
-        source_objects=[
-            f"{EXPORT_PREFIX}/all_namespaces/kind_{EXPORT_COLLECTION_ID}"
-            f"/all_namespaces_kind_{EXPORT_COLLECTION_ID}.export_metadata"
-        ],
```

Comment:
Why it should be removed? is it not a part of the example?

Reply:
It's remnant from previous (deprecated) way of using the operator. You should not mix "source_objects" and "table_resource". 

We coul probably add corresponding "source_uris" in the "table_resources" but since I did not want to be "wrong" and make some mistake there, I prefered to remove it (table_resources has many more optional fields that can be used).

Reply:
great. can be done in a followup PR if needed.